### PR TITLE
Made changes to how the arguments are handled

### PR DESCRIPTION
**Pull request by thilton-oddball**
Made changes to how the arguments are handled, with module name now being passed with a flag. Also updated tests to reflect these changes.

### CODE REVIEW

File: spec/spec_helper.rb [hunk 1]
```diff
@@ -11,6 +11,6 @@
 
   config.expect_with :rspec do |c|
     c.syntax = :expect
-    c.max_formatted_output_length = nil
+    # c.max_formatted_output_length = nil
```
Comment (LindseySaari) [inline spec/spec_helper.rb:14]: Do we want to remove this line?

File: spec/unit/module/controller_spec.rb [hunk 2]
```diff
@@ -5,12 +5,12 @@
 RSpec.describe Vtk::Commands::Module::Controller do
   it 'executes `module controller` command successfully' do
     output = StringIO.new
-    name = 'foo'
-    options = {}
+    name = 'bar'
```
Comment (LindseySaari) [inline spec/unit/module/controller_spec.rb:8]: Should we test the scenario where the module name is not provided. Looks like it's a required option

File: spec/spec_helper.rb (hunk 1 above)
Comment (thilton-oddball) [reply spec/spec_helper.rb:14]: Yes, I was going back and forth on that during testing. I'll pull it out.

File: spec/unit/module/controller_spec.rb (hunk 2 above)
Comment (thilton-oddball) [reply spec/unit/module/controller_spec.rb:8]: Good call, I will add that in.

**Pull request merged**